### ISSUE TITLE
tf.nn.softmax_cross_entropy_with_logits_v2

### ISSUE THREAD

**Issue by Elody-07** [imransai/Depth-Coefficients-from-Single-Image#5]
Thanks for providing full code. I noticed that you use tf.nn.softmax_cross_entropy_with_logits_v2 as loss function. But this function seems to fit one-hot labels whereas yours are three-hot. Is this a problem in your training?
